yocto/runqemu-bitbake-test.sh

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2017 Pelagicore AB
```

Comment:
the tests can run on any variant. So far from what i've tested, you need to have the images already build to test bitbake. I wrote runqemu, since the only time the tests will be run is when qemu is being tested weekly. However Qemu doesnt need to be ran for bitbake tests. I believe i should modify the name..

Reply:
Then I think it is best to change the filename so future users aren't confused and think it can only run on Qemu, regardless of how we run the tests now.

Reply:
But with that said it would be really good to add an explanation in the file on how it is currently run.